src/main/java/duke/ui/Message.java

```diff
@@ -0,0 +1,30 @@
+package duke.ui;
+
+import duke.task.TaskList;
+
+public class Message {
+    public static final String MESSAGE_WELCOME = "Hello! I'm Duke" + Ui.LS + "What can I do for you?";
+    public static final String MESSAGE_EXIT = "Bye. Hope to see you again soon!";
+    public static final String MESSAGE_INDEX = "Here are the tasks in your list:";
+    public static final String MESSAGE_FIND = "Here are the matching tasks in your list:";
+    public static final String MESSAGE_ADD = "Got it. I've added this task:";
+    public static final String MESSAGE_DELETE = "Noted. I've removed this task:";
+    public static final String MESSAGE_DONE = "Nice! I've marked this task as done:";
+    public static final String MESSAGE_NO_TASKS_IN_LIST = "There are currently no tasks in your list.";
+    public static final String MESSAGE_LOADING_ERROR = "An empty data file duke.txt is "
+            + "created in the current directory.";
+    public static final String MESSAGE_NO_MATCHING_TASKS = "There are no matching tasks for the key word.";
```

Comment:
If this class is entirely static, I would consider making it a `final` class such that it cannot be extended unintentionally.